sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala

```diff
@@ -46,8 +46,8 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
 
   def getString(key: K, default: Option[String]): Option[String] = {
```

Comment:
We need Option[String] in result, "def getString(key: K): String" is not possible.
We can add a new function "def getString(key: K): Option[String]" ???